inventory/inventory_data.go

```diff
@@ -75,7 +76,8 @@ func (id *InventoryDataRunner) Get() (client.InventoryData, error) {
 
 	idec := NewInventoryDataDecoder()
 	for _, t := range tools {
-		cmd := id.cmd.Command(t)
+		cmd, logger := id.cmd.Command(t).LogStderr()
```

Comment:
For now only added to the inventory capturing. 
You reckon this should be added every single place we call a script?

Reply:
I think "assume everywhere unless it doesn't make sense" is a good strategy.

Reply:
With this approach, we will have to be explicit about where it is added though, so I will just go through our calls manually, and add as I see fit.

Reply:
Indeed. This is why I initially wanted to have it as a property of the `system.Cmd` struct. But since this turned out pretty tricky, we'll have to do the explicit approach.

Reply:
Added to `identity` and `inventory` fetching.

Other places this has been added on a per command basis it looks to me. Like the update modules have something similar in place, where they write to a log struct, which wraps the output.

Do you want to unify this, or should we keep it as it is?

I know the natural answer is unify it, but this will change quite a lot of our output, so the logging will be quite different. I favour this approach, but not sure if you think it too invasive?

Reply:
I also favor it. This will be minor-release material, so I think it's ok to change it. One could argue that it's a behavior change, but logs should be allowed to change, IMHO.